libkbfs/folder_block_ops.go

```diff
@@ -2585,14 +2575,36 @@ func (fbo *folderBlockOps) getDeferredWriteCountForTest(lState *lockState) int {
 	return len(fbo.deferredWrites)
 }
 
+func (fbo *folderBlockOps) updatePointer(kmd KeyMetadata, oldPtr BlockPointer, newPtr BlockPointer) {
+	updated := fbo.nodeCache.UpdatePointer(oldPtr.Ref(), newPtr)
+	if !updated {
+		return
+	}
+	// Prefetch the new reference, but only if it already exists in the block
```

Comment:
Hmm, this check only seems useful if the node cache (collective across all TLFs) is much bigger than the block cache, right?  I feel like there's a chance the check could be harmful, in the case where a block for a Node is prefetched, but then we get an update for it before the block returns.  In that case, we'll never prefetch the updated block.  So I think I vote to skip this check.  What do you think?